configure.ac

```diff
@@ -182,7 +182,7 @@ AC_ARG_ENABLE([integration],
         [build and execute integration tests])],,
     [enable_integration=no])
 AS_IF([test "x$enable_integration" = "xyes"],
-      [ERROR_IF_NO_PROG([tpm_server])
+      [ERROR_IF_NO_PROG([swtpm])
```

Comment:
The test in [l. 199](https://github.com/tpm2-software/tpm2-tss/blob/30afabfc81496bc8a225e67b4d090ba1e169a22a/configure.ac#L199) for the mssim TCTI module needs to be changed to swtpm as well.

Reply:
Thanks, solved this by the latest changes.